.gitignore

```diff
@@ -2,6 +2,8 @@
 #
 # gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
 
+*.DS_Store
```

Comment:
Please include this in a separate PR, this change is not related to "Carthage readme" update